src/kyc.ts

```diff
@@ -51,4 +64,55 @@ export class KycApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
+
+  /*
+   * Function that takes a Marqeta User token and gets a list of the KYC
+   * results for that User.
+   */
+  async userResults(search :{
+    token ? : string,
+      count ? : number,
+      startIndex ? : number,
+      fields ? : string[],
+      sortBy ? : string,
```

Comment:
These last four lines have a little too much indentation... 

And why is there a space before, and after, the `?`? That seems different... what's up?

Reply:
cut-n-paste carelessness